draft-brw-sconepro-rate-policy-discovery.md

```diff
@@ -883,6 +885,24 @@ This example assumes that the Network Access Server (NAS) embeds both Remote Aut
 ~~~~~
 {: #radius-ex title="An Example of RADIUS NRLP Exchanges"}
 
+# Alternative/Complementary Mechanisms
+
+In the event of bottlenecks in a network, there are other mechanisms that provide information or help to reserve resources. These can be used within the bottleneck network or, in some cases, across network boundaries. The following sections give examples of such mechanisms and provide background information.
+
+## L4S
+
+Low Latency, Low Loss, and Scalable Throughput (L4S) is a function defined in {{RFC9330}} to avoid queuing at bottlenecks by capacity-seeking congestion controllers of senders. Thus, it addresses the investigated use case of this document, which considers rate limiting, which typically involves queuing discipline at the rate limiting bottleneck. If the network and service support L4S, the use of Explicit Congestion Notification (ECN) is the measure used to inform the network protocol and/or service endpoint in use of impending congestion.
```

Comment:
```suggestion
Low Latency, Low Loss, and Scalable Throughput (L4S) is an architecture defined in {{RFC9330}} to avoid queuing at bottlenecks by capacity-seeking congestion controllers of senders. L4S support addresses the investigated use case of this document, which considers rate limiting, which typically involves queuing discipline at the rate limiting bottleneck. If all involved elements (UE, network, and service) support L4S, the use of Explicit Congestion Notification (ECN) provides the measure used to inform the network protocol and/or service endpoints in use of impending congestion. Congestion detection and reaction may require some few RTTs, though to adjust to the network forwarding conditions.
```